packages/arb-token-bridge-ui/tests/e2e/specs/switchNetworks.cy.ts

```diff
@@ -0,0 +1,136 @@
+        // to view the correct list of networks (and not testnets), first navigate to mainnet
+        cy.changeMetamaskNetwork('mainnet').then(() => {
+          // first wait for Low Balance Popup and close that
+          cy.findByRole('button', { name: /Go to bridge/i })
+            .should('be.visible')
+            .click({ scrollBehavior: false })
+
+          cy.findByRole('button', { name: /Selected Network : /i })
+            .should('be.visible')
+            .click({ scrollBehavior: false })
+
+          cy.findByRole('button', { name: /Switch to Arbitrum One/i }).should(
+            'be.visible'
+          )
+
+          //close the dropdown
+          cy.findByRole('button', { name: /Selected Network : /i })
+            .should('be.visible')
+            .click({ scrollBehavior: false })
+        })
+      })
+
+      it('should change network to Arbitrum One successfully', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Arbitrum One/i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.allowMetamaskToAddAndSwitchNetwork().then(() => {
+          cy.findByRole('button', {
+            name: /Selected Network : Arbitrum One/i
+          }).should('be.visible')
+        })
+      })
+
+      it('should change network to Arbitrum Nova successfully', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Arbitrum Nova/i }).click({
+          scrollBehavior: false
+        })
+
+        cy.allowMetamaskToAddAndSwitchNetwork().then(() => {
+          cy.findByRole('button', {
+            name: /Selected Network : Arbitrum Nova/i
+          }).should('be.visible')
+        })
+      })
+
+      it('should change network to Ethereum mainnet successfully', () => {
+        cy.findByRole('button', { name: /Selected Network : /i })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('button', { name: /Switch to Mainnet/i }).click({
+          scrollBehavior: false
+        })
+
+        cy.allowMetamaskToSwitchNetwork().then(() => {
+          cy.findByRole('button', {
+            name: /Selected Network : Mainnet/i
+          }).should('be.visible')
+        })
+      })
+    })
+
+    context('Test Networks list in Wrong Network UI', () => {
+      it('should show wrong network UI', () => {
+        cy.changeMetamaskNetwork('kovan').then(() => {
+          cy.findByText(/Oops! You’re connected to a wrong network./i).should(
+            'be.visible'
+          )
+        })
+      })
+
+      it('should allow Network change from wrong network UI list', () => {
+        cy.changeMetamaskNetwork('kovan').then(() => {
```

Comment:
kovan has been deprecated for a while and is removed from the test network list on newer versions of metamask
can we use sepolia instead?

and the switch buttons in this test should be going to goerlis